libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -61,4 +67,8 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  isFinished(bookId: string) {
```

Comment:
This method is not required. We have property finished for `ReadingListItem` which we can use in HTML to check for book finished or not.

Reply:
Changes are done.